tests/test_cleanup_marathon_jobs.py

```diff
@@ -135,6 +135,7 @@ def test_delete_app(self):
                 cluster='fake_cluster',
                 line=expected_log_line,
             )
+            assert mock_send_sensu_event.call_count == 3
```

Comment:
Checking that `setup_marathon_job.123` is one of the calls would give more useful test-failure messages and be more robust if we have to add a 4th thing... but this works.